hr_calendar_multiweek/models/resource_calendar.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+from odoo import api, fields, models
+
+
+class ResourceCalendar(models.Model):
+    _inherit = 'resource.calendar'
+
+    @api.multi
+    def _get_day_attendances(self, day_date, start_time, end_time):
+        res = super()._get_day_attendances(day_date, start_time, end_time)
+        return res.filtered(lambda r: r._check_week(day_date))
+
+
+class ResourceCalendarAttendance(models.Model):
+    _inherit = "resource.calendar.attendance"
+
+    calendar_week_number = fields.Integer(default=1)
+    week_number = fields.Integer(default=1)
```

Comment:
@etobella Could you please add a `help` to this fields or explain their difference better in the readme? I think it is not clear at first sight what they are meant to be used for and it may help other people review this module and get it merged. :wink: :wink: